build/generate.mk

```diff
@@ -73,10 +73,22 @@ generate-controller: generate-controller-gen-installed ## Generates the CRDs for
 	controller-gen object:headerFile=./boilerplate.go.txt paths=./pkg/controller/api/...
 	controller-gen crd paths=./pkg/controller/api/... output:crd:dir=./deploy/Chart/crds/radius
 
+.PHONY: generate-cleanup
+generate-cleanup: ## Deletes all generated code.
+	@echo "$(ARROW) Deleting generated code..."
+	find . -type f -name 'zz_*.go' ! -name 'zz_*.deepcopy.go' -delete
+	@echo "$(ARROW) Done."
+
+.PHONY: generate-fix-version
+generate-fix-version: ## Fixes the module version in generated files.
+	@echo "$(ARROW) Fixing module version..."
+	grep -rl --include \zz_*.go 'moduleVersion = "v0.1.0"' | xargs -r sed -i "s/moduleVersion = \"v0.1.0\"/moduleVersion = \"v$(AUTOREST_MODULE_VERSION)\"/"
+	@echo "$(ARROW) Done."
+
 .PHONY: generate-genericcliclient
 generate-genericcliclient: generate-node-installed generate-autorest-installed
 	@echo "$(AUTOREST_MODULE_VERSION) is module version"
-	autorest pkg/cli/clients_new/README.md --tag=2023-10-01-preview
+	autorest pkg/cli/clients_new/README.md --tag=2023-10-01-preview && rm pkg/cli/clients_new/generated/go.mod
```

Comment:
Why do we need to run `rm pkg/cli/clients_new/generated/go.mod` command after running the autorest command?

Reply:
Is there a way to instruct the generator to not produce go.mod instead of deleting it?